api/server/server.go

```diff
@@ -149,7 +149,6 @@ func (s *Server) makeHTTPHandler(handler httputils.APIFunc) http.HandlerFunc {
 }
 
 // InitRouter initializes the list of routers for the server.
-// This method also enables the Go profiler if enableProfiler is true.
```

Comment:
I think it still enables the Go profiler, only no longer has an argument to enable it, correct?

Reply:
Sure `InitRouter` calls `createMux`. `createMux` sets `debug.NewRouter()` to `Server.routers`. So it enables the profiler indirectly.

Reply:
We could probably keep everything but: `if enableProfiler is true`

It would be nice if this was refactored so that the caller passes in the debug router, though.

Reply:
Ok, I will keep the sentence "This method also enables the Go profiler".